src/server/models/news/router.js

```diff
@@ -0,0 +1,47 @@
+const router = require('express').Router();
+
+const path = require('path');
+const fs = require('fs');
+const util = require('util');
+const News = require('./model');
+
+const readFileAsync = util.promisify(fs.readFile);
+const writeFileAsync = util.promisify(fs.writeFile);
+
+const storageFilePath = path.resolve(__dirname, '../data.json');
+
+router.route('/api/my-news/:id?')
```

Comment:
get = get particular news
put = update particular news
delete = delete particular news

Reply:
you forgot about put

Reply:
fixed